runtime/src/lib.rs

```diff
@@ -3,24 +3,80 @@
 #![no_std]
 
 mod mailbox;
+mod verify;
 
 use mailbox::Mailbox;
 
 use caliptra_common::cprintln;
 use caliptra_drivers::{caliptra_err_def, CaliptraResult};
-use caliptra_registers::mbox::enums::MboxStatusE;
-
-use core::mem::size_of;
+use caliptra_registers::{mbox::enums::MboxStatusE, soc_ifc};
+use zerocopy::{AsBytes, FromBytes};
 
 caliptra_err_def! {
     Runtime,
     RuntimeErr
     {
         // Internal
         InternalErr = 0x1,
+        UnimplementedCommand = 0x2,
+        InsufficientMemory = 0x3,
+    }
+}
+
+pub enum CommandId {
+    FirmwareLoad = 0x46574C44,     // "FWLD"
```

Comment:
Note: in "hexdump -C", this little-endian value will appear as "DLWF"; do we care?

IIRC, convention with FourCC is to ensure they are human-readable when serialized.

Reply:
https://github.com/chipsalliance/caliptra-sw/issues/275